tests/tasks/assert_bond_options.yml

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: "** TEST check bond settings"
+  command: cat
+    /sys/class/net/{{ controller_device }}/bonding/'{{ item.key }}'
+  register: result
+  until: "'{{ item.value }}' in result.stdout"
+  loop:
+    - {key: 'mode', value: '802.3ad'}
+    - {key: 'ad_actor_sys_prio', value: '65535'}
+    - {key: 'ad_actor_system', value: '00:00:5e:00:53:5d'}
+    - {key: 'ad_select', value: 'stable'}
+    - {key: 'ad_user_port_key', value: '1023'}
+    # wokeignore:rule=slave
+    - {key: 'all_slaves_active', value: '1'}
+    - {key: 'downdelay', value: '0'}
+    - {key: 'lacp_rate', value: 'slow'}
+    - {key: 'lp_interval', value: '128'}
+    - {key: 'miimon', value: '110'}
+    - {key: 'num_grat_arp', value: '64'}
+    - {key: 'resend_igmp', value: '225'}
+    - {key: 'updelay', value: '0'}
+    - {key: 'use_carrier', value: '1'}
+    - {key: 'xmit_hash_policy', value: 'encap2+3'}
```

Comment:
Is it possible to assign this as a variable in `tests/playbooks/tests_bond_options.yml`?

Reply:
It is possible.

Reply:
Then please do so. It seems to me, that it will help making the playbooks easier to understand. What do you think? It should also reduce the amount of needed task files.

Reply:
Yes, we do not need to keep some redundant task files.